packages/near-api-js/test/account.test.js

```diff
@@ -307,4 +307,10 @@ describe('with deploy contract', () => {
         });
         expect(result).toEqual('hello trex');
     });
+
+    test('get total stake balance', async() => {
```

Comment:
Is it possible to have coverage for delegating some stake and getting the updated balance?